version.props

```diff
@@ -40,15 +40,17 @@
     <BuildNumberSuffix>$(_BuildNumberShortDate)-$(VersionSuffixBuildOfTheDayPadded)</BuildNumberSuffix>
 
     <FileRevisionVersion>$(_BuildNumberShortDate)</FileRevisionVersion>
+    <VersionSuffix>$(PreReleaseLabel)-$(BuildNumberSuffix)</VersionSuffix>
```

Comment:
Would changing the dash to a period resolve the format difference in releases between AspNetCoreRuntime.3.0.x64.3.0.0-preview-19081-0488 and Microsoft.Extensions.Logging.3.0.0-preview5.19227.9 ?

Reply:
Yes it would resolve the format difference, but it would create more problems. See https://github.com/aspnet/AspNetCore/issues/4339

Reply:
That issue was written before we moved to include the preview number in our versions. So, we could make the switch when we next update the branding in this repo. Suggest reopening #4339.

Reply:
Re opened. I'll bring it up during our area-platform triage. It's not really a build team decision to do this or not because this impacts the appearance and behavior of the product, so I won't attempt to address that in this PR.